### DISCUSSION TITLE
Export gif

### DISCUSSION THREAD

**Discussion by Lishan997** [jgraph/drawio#3750] [Q&A]
I created animated digram (Flow animation). But i am not allowed to export it as a gif , there is no feature export as a gif in export file section, what are the alternationve solution for that problem. 

I tried many ways, created HTML and tried to convert it to gif none of works are not successfull

**Comment by davidjgraph** [comment]
There is no export to git or other kind of animation built-in. You'd have to use a third-party tool.

**Comment by davidjgraph** [comment]
https://github.com/jgraph/drawio/issues/3739
